app-text/nfoview/nfoview-1.28.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=no
```

Comment:
why is this a no?

Reply:
seems like it should be `bdepend`

Reply:
Please do not comment unless you understand and have verified the author's assumption. If you check the `setup.py`, you will see that they rely solely on `distutils` without even an optional attempt at `setuptools`.

Reply:
ah, you are correct. My apologies.